aqua/input-leap/Portfile

```
@@ -0,0 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        input-leap input-leap 198779cf9e798aac6ad41427e32d031305736de3
+version             20230416
+revision            0
+categories          aqua net sysutils
+platforms           darwin
+license             GPL-2
+maintainers         iki.fi:koston
+description         share a keyboard and mouse over the network
+long_description    ${name} shares a keyboard, mouse, and clipboard over the network.\
+                    It supports multiple operating systems, including macOS,\
+                    Linux, FreeBSD, OpenBSD, and Windows. It is a fork of barrier,\
+                    which in turn is a fork of synergy.
+
+checksums           rmd160  c6da02aaf782727da7cfd62f2fc11eac41da3834 \
+                    sha256  9f893bfa8719a62acb7f7012847a40988d5ab6ec8f75e78881d68988a96ea4e0 \
+                    size    5171389
+
+patchfiles          set-cmake-revision.patch \
+                    remove-gtest.patch
+patch.pre_args      -p1
+
+compiler.cxx_standard 2014
+
+depends_build-append \
+    port:pkgconfig \
+    port:gtest \
+    port:ghc-filesystem
+
+depends_lib-append \
+    path:lib/libssl.dylib:openssl
+
+configure.args-append \
+    -DINPUTLEAP_BUILD_INSTALLER=OFF \
+    -DINPUTLEAP_BUILD_TESTS=OFF
+
+variant debug {
+    cmake.build_type Debug
+    configure.optflags "-O0"
+}
+
+# the gui requires metal
+if { ${os.major} < 16 } {
```

Check `${os.platform}` before checking `${os.major}`:
```suggestion
if {${os.platform} eq "darwin" && ${os.major} < 16} {
```